docs/design-ux/editor.md

```diff
@@ -0,0 +1,37 @@
+# Editor
+When a user has a document selected, that file is open for editing in an editor. The name of the document is displayed above the document and is editable (editing renames the document).
+
+An icon near the document's name indicates one of the following statuses:
+* the document is presented exactly as it appears on disk
```

Comment:
I wonder if

> all changes to the document have been merged into the version on disk

Is too complicated an idea to map cleanly to an icon that's going to exist on every platform. 

> there are unsaved changes to the document

could map to an :writing_hand: icon

> the document is presented exactly as it appears on disk

could map to :heavy_check_mark: 

emoji could be a good way to bound which icons we use within the app. Partly because we don't currently have the resources to author good icon sets, but mainly because they have meaning that people already understand.

Reply:
how about some kind of refresh/🔁 type icon, to indicate that there's something on disk that isn't in your editor? do you think having ✍️ and 🔁 show up next to each other at the same time is too bad of a way to indicate that there are unsaved changes in the editor *and* unloaded changes on disk at the same time?